recipes/opencolorio/all/conandata.yml

```
@@ -9,6 +12,16 @@ sources:
     url: "https://github.com/AcademySoftwareFoundation/OpenColorIO/archive/v1.1.1.tar.gz"
     sha256: "c9b5b9def907e1dafb29e37336b702fff22cc6306d445a13b1621b8a754c14c8"
 patches:
+  "2.3.0":
+    - patch_file: "patches/2.3.0-0001-fix-cmake-source-dir-and-targets.patch"
+      patch_description: "use cci package, use PROJECT_BINARY_DIR/PROJECT_SOURCE_DIR"
+      patch_type: "conan"
+    - patch_file: "patches/2.3.0-0002-fix-pystring.patch"
+      patch_description: "fix include path for pystring"
+      patch_type: "conan"
```

This fix is reported upstream?

---

Good point. Just about to start the discussion and created a proposed MR: https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/1900 and will follow up in their Slack channel about those changes.

---

There is another MR in the works that should solve the problem as we haven't been the only ones running into problems where I give feedback on now: https://github.com/remia/OpenColorIO/pull/52

---

https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/1901 has now been raised. Tested it locally by adding a zip of the PR branch as a separate version and it build without the patch.

---

@franramirez688 Any chance to move forward with this?

The upstream seems to be on a good track btw. and got the first approval.

---

Hi @irieger 
Sorry for the delay. Thanks a lot for the info 😁 
Then, we should put that info in the conandata.yml:

```suggestion
      patch_type: "portability"
      patch_source: "https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/1901"
```